src/main/scala/io/phdata/retirementage/loadgen/LoadGeneratorConfig.scala

```diff
@@ -28,5 +28,6 @@ class LoadGeneratorConfig(args: Array[String]) extends ScallopConf(args) {
   val dimName  = opt[String]("dim-name", required = false, default = Some("dimloadtest"))
   val subName  = opt[String]("subdim-name", required = false, default = Some("subloadtest"))
 
+  val testKudu = opt[Boolean]("test-kudu", required = false, default = Some(false))
```

Comment:
How about adding an 'output-format' enum option? Options could be 'parquet' or 'kudu' for now. (avro if it works)

Reply:
ok I see what's going on here. This test-kudu should definitely go in a unit test.